ws/libcrochess/inc/cc_str_utils.h

```diff
@@ -384,7 +384,7 @@ char const * cc_str_contains_char( char c,
 
 /**
     Function returns a string pointer, by traversing a given string,
-    and either skiping over filtered characters, or stoping at first of those.
+    and either skyping over filtered characters, or stopping at first of those.
```

Comment:
Hello, and thank you so much for your contribution.

Before I merge I'd like to ask you what's "skyping"?
It should be "skipping", right?

Best regards,
Mario